openlibrary/templates/books/mobile_carousel.html

```diff
@@ -49,8 +49,8 @@
       <a class="mybooks mobile nopadding" href="$(url)">
         $if cover_url:
           <img class="bookcover mobile $(last_book)" loading="lazy"
-            title="$('%s%s'%(title,byline))"
-            alt="$('%s%s'%(title,byline))"
+            title="$_('%(title)s%(byline)s', title=title, byline=byline)"
+            alt="$_('%(title)s%(byline)s', title=title, byline=byline)"
```

Comment:
Ah there's nothing to actually translate here! So can probably just replace with:

```suggestion
            title="$title$byline"
            alt="$title$byline"
```